### PR TITLE
[API-1856] Add BGS gem tests for Power of Attorney End points

### PR DESCRIPTION
**Pull request by templeman15**
## Description of change
This PR adds specs for Claimant and VeteranRecord services using the VCR gem.

## Original issue(s)
[Add BGS gem tests for Power of Attorney End points](https://vajira.max.gov/browse/API-1856)

### CODE REVIEW

**Review by bastosmichael — APPROVED** [review]
LGTM but I want to get @charleystran's take on this as well since he was the original author of the gem.

File: lib/bgs/services/vet_record.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def update_birls_record(ssn:, poa_code:)
           "birlsUpdateInput": {
             "CLAIM_NUMBER": ssn,
             "PAYEE_NUMBER": '00',
-            "POWER_OF_ATTY_CODE1": poa_code.first,
+            "POWER_OF_ATTY_CODE1": poa_code[0],
```
Comment (charleystran) [inline lib/bgs/services/vet_record.rb:31]: good catch since active support is not guaranteed

**Review by charleystran — COMMENTED** [review]
Can you bump the version in the gemspec file?

**Comment by templeman15** [comment]
> Can you bump the version in the gemspec file?

Bumped and pushed!

**Review by charleystran — APPROVED** [review]

**Pull request merged**